sdk/iotcentral/Azure.ResourceManager.IotCentral/tests/IotCentralLifeCycleTests.cs

```diff
@@ -0,0 +1,200 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Could you please follow our track 2 sdk test pattern? 

1. First, we will define a [base test class](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cdn/Azure.ResourceManager.Cdn/tests/CdnManagementTestBase.cs) which inherits from `ManagementRecordedTestBase` and its name will be in the form `{RP}ManagementTestBase`. And in this base class we defined the `Client` field and used the attribute `SetUp` to construct it. 
2. Then we will have two cs files to test different methods of the one resource separately. In other words, the file named `{Resource}CollectionTests` test the methods in `{Resource}Collection` and `{Resource}OperationsTests` test the methods in `{Resource}Resource`. Both of them are the derived classed of the base test class. Here is the [example](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cdn/Azure.ResourceManager.Cdn/tests/Scenario/AfdCustomDomainCollectionTests.cs) of collection test in Cdn.
3. One more thing to say is, we use these [two attributes](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cdn/Azure.ResourceManager.Cdn/tests/Scenario/AfdCustomDomainCollectionTests.cs#L37-L38) to mark test instead of `AsyncOnly`. You can just test the asynchronous methods and our test framework will automatically help you to test the corresponding synchronous methods.